### ISSUE TITLE
Tuning model.tune with custom parameters range

### ISSUE THREAD

**Issue by Timmy-Eink** [ultralytics/ultralytics#17629]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

When I use space for tuning, each iteration uses the same parameters. However, when I remove space for tuning, it returns to normal. If I want to add dropout or mixup to the tuning parameters, how should I do that?

my code :
```
from ultralytics import YOLO
import os
import torch

os.environ["CUDA_LAUNCH_BLOCKING"] = "1"
os.environ["TORCH_USE_CUDA_DSA"] = "1"
os.environ["OMP_NUM_THREADS"] = "1"
os.environ["NCCL_NET"] = "Socket"
os.environ["TORCH_CPP_LOG_LEVEL"] = "1"
os.environ["NCCL_DEBUG"] = "INFO"
torch.cuda.empty_cache()


if __name__ == "__main__":
    model = YOLO(model="/home/timmy/ultralytics/weights/yolo11n-seg.pt", task="segment")

    model.tune(
        data="/home/timmy/ultralytics/data/TM2-keyence.yaml",
        iterations=100,
        epochs=300,
        patience=50,
        optimizer="auto",
        project="/data/TM2/241118-640-keyence20",
        imgsz=640,
        cache="disk",
        overlap_mask=True,
        batch=16,
        device=1,
        space={
            "dropout": (0, 0.8),
            "degrees": (0, 30),
            "mixup": (0, 0.3),
        },
    )

```

Is that correct?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Timmy-Eink, thank you for reaching out and showing interest in Ultralytics 🚀! This is an automated response, and an Ultralytics engineer will assist you soon.

We recommend visiting our [Docs](https://docs.ultralytics.com) if you're new to Ultralytics. Here, you'll find detailed instructions and examples for [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage that could help you.

For your custom training ❓ question, please ensure you're providing as much detail as possible. Your shared code snippet is a great start! To further assist us, you could also include dataset image examples and training logs. Additionally, check out our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to ensure you're getting the most out of your model training.

If this issue behaves like a 🐛 Bug and persists even after following the best practices, providing a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) could greatly assist our team in pinpointing the issue.

Join our Ultralytics community to connect with others! For real-time chat, visit our [Discord](https://ultralytics.com/discord) 🎧. For detailed discussions, browse our [Discourse](https://community.ultralytics.com). Or, engage with others on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

To make sure you're on the latest software, upgrade your `ultralytics` package along with all its [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment using [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

For operating YOLO smoothly, you can utilize the following verified environments, which come with all dependencies (including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) pre-installed):

- **Notebooks** with GPU access: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud**: See our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon**: Refer to our [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: Follow our [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

Keep an eye on this badge to check if all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing, ensuring that YOLO operates correctly across macOS, Windows, and Ubuntu. These tests run every day and with every code commit.

Thank you for your inquiry, and we look forward to assisting you further! 😊
